services/121-service/src/user/user.service.ts

```diff
@@ -520,14 +520,13 @@ export class UserService {
     return user;
   }
 
-  public async findByUsernameOrThrow(username: string): Promise<UserEntity> {
+  public async findByUsernameOrThrow(
+    username: string,
+    relations: FindOptionsRelations<UserEntity> | undefined,
```

Comment:
```suggestion
    relations?: FindOptionsRelations<UserEntity>,
```

Small nitpick, but I find this generally makes it easier to, at a glance, check which parameters are optional.